lib/dev.ts

```diff
@@ -236,3 +253,21 @@ async function generateDevJson(src: string, config: BaseConfig): Promise<DevJson
 
   return devJson;
 }
+
+export function buildGatewayObject(commandGateway, configGateway) {
+	// Gateway logic:
+	// if --no-gateway is provided (or commandGateway = false), gateway should be disabled ("");
+	// if -g is true, gateway is enabled, takes bundleUrl and tagName from `bos.config.json`
```

Comment:
-g would be true by default, we can remove "gateway" from our cli actually

Only thing would be if user provides --no-gateway, which will disable it